### PR TITLE
Add new flag aliases(apple)

### PR DESCRIPTION
**Pull request by waxion03**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Persistence flags should be deprecated and be an alias of the new flags. 

## Related PRs or issues

List related PRs and other issues.

### CODE REVIEW

File: AppCenter/AppCenter/MSConstants+Flags.h [hunk 1]
```diff
@@ -5,7 +5,10 @@
...
 typedef NS_OPTIONS(NSUInteger, MSFlags) {
   MSFlagsNone = (0 << 0),                // => 00000000
-  MSFlagsPersistenceNormal = (1 << 0),   // => 00000001
-  MSFlagsPersistenceCritical = (1 << 1), // => 00000010
-  MSFlagsDefault = MSFlagsPersistenceNormal
+  MSFlagNormal = (1 << 0),               // => 00000001
+  MSFlagCritial = (1 << 1),              // => 00000010
+  MSFlagsPersistenceNormal = MSFlagNormal,
+  MSFlagsPersistenceCritical = MSFlagCritial,
+  MSFlagsPersistenceNone = MSFlagsNone,
```
Comment (MatkovIvan) [inline AppCenter/AppCenter/MSConstants+Flags.h]: Remove this

File: AppCenter/AppCenter/MSConstants+Flags.h [hunk 2]
```diff
@@ -5,7 +5,10 @@
...
+  MSFlagsDefault = (MSFlagNormal | MSFlagsPersistenceNormal)
```
Comment (MatkovIvan) [inline AppCenter/AppCenter/MSConstants+Flags.h]:
```suggestion
  MSFlagsDefault = MSFlagNormal
```

File: Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift [hunk 3]
```diff
@@ -14,9 +14,9 @@ class MSAnalyticsViewController: UITableViewController, AppCenterProtocol {
     var flags: MSFlags {
       switch self {
       case .Normal:
-        return [.persistenceNormal]
+        return [.MSFlagNormal]
```
Comment (MatkovIvan) [inline Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift]: Maybe `.normal`?
It doesn't seem correct

**Comment by MatkovIvan** [comment]
@waxion03 Fix CI build

File: AppCenter/AppCenter/MSConstants+Flags.h [hunk 4]
```diff
@@ -5,7 +5,10 @@
 
 typedef NS_OPTIONS(NSUInteger, MSFlags) {
   MSFlagsNone = (0 << 0),                // => 00000000
-  MSFlagsPersistenceNormal = (1 << 0),   // => 00000001
-  MSFlagsPersistenceCritical = (1 << 1), // => 00000010
-  MSFlagsDefault = MSFlagsPersistenceNormal
+  MSFlagsNormal = (1 << 0),               // => 00000001
+  MSFlagsCritial = (1 << 1),              // => 00000010
+  MSFlagsPersistenceNormal = MSFlagsNormal,
```
Comment (MatkovIvan) [inline AppCenter/AppCenter/MSConstants+Flags.h]: Is there a way to mark it as deprecated?

**Comment by jaelim-ms** [comment]
I think it would be better to add `.vscode` in `.gitignore`.

File: AppCenter/AppCenter/MSConstants+Flags.h [hunk 5]
```diff
@@ -5,7 +5,10 @@
 
 typedef NS_OPTIONS(NSUInteger, MSFlags) {
   MSFlagsNone = (0 << 0),                // => 00000000
-  MSFlagsPersistenceNormal = (1 << 0),   // => 00000001
-  MSFlagsPersistenceCritical = (1 << 1), // => 00000010
-  MSFlagsDefault = MSFlagsPersistenceNormal
+  MSFlagsNormal = (1 << 0),               // => 00000001
+  MSFlagsCritial = (1 << 1),              // => 00000010
```
Comment (clpolet) [inline AppCenter/AppCenter/MSConstants+Flags.h]:
```suggestion
  MSFlagsCritical = (1 << 1),              // => 00000010
```

File: AppCenter/AppCenter/MSConstants+Flags.h (hunk 4 above)
Comment (waxion03) [reply AppCenter/AppCenter/MSConstants+Flags.h]: sure！

**Comment by annakocheshkova** [comment]
We should update this PR once all the other feature PRs are merged and merge after them.

File: AppCenter/AppCenter/MSConstants+Flags.h [hunk 6]
```diff
@@ -5,7 +5,10 @@
...
 typedef NS_OPTIONS(NSUInteger, MSFlags) {
   MSFlagsNone = (0 << 0),                // => 00000000
-  MSFlagsPersistenceNormal = (1 << 0),   // => 00000001
-  MSFlagsPersistenceCritical = (1 << 1), // => 00000010
-  MSFlagsDefault = MSFlagsPersistenceNormal
+  MSFlagsNormal = (1 << 0),               // => 00000001
+  MSFlagsCritical = (1 << 1),              // => 00000010
+  MSFlagsPersistenceNormal DEPRECATED_MSG_ATTRIBUTE("please use MSFlagsNormal") = MSFlagsNormal,
+  MSFlagsPersistenceCritical DEPRECATED_MSG_ATTRIBUTE("please use MSFlagsCritical") = MSFlagsCritical,
+  MSFlagsPersistenceNone DEPRECATED_MSG_ATTRIBUTE("please use MSFlagsNone") = MSFlagsNone,
```
Comment (jaelim-ms) [inline AppCenter/AppCenter/MSConstants+Flags.h]: `MSFlagsPersistenceNone` is newly added to the PR and you are deprecating it. I don't think we need this one, please remove.

Comment (jaelim-ms) [reply AppCenter/AppCenter/MSConstants+Flags.h]:
```suggestion
```

**Pull request merged**